### ISSUE TITLE
The Editor should be focused once a note is created

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#17]
```
When i create a new note, i'd like to have the editor focused to be
able to immediatly start typing without having to 1st select the
editor
```

Original issue reported on code.google.com by `thibaut....@gmail.com` on 4 Mar 2009 at 12:42

**Comment by GoogleCodeExporter** [comment]
```
This is a good idea.  I have added the feature and should be available in the 
next
release.
```

Original comment by `matt.rasmus@gmail.com` on 9 Mar 2009 at 9:03
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
great, thanks!
```

Original comment by `thibaut....@gmail.com` on 10 Mar 2009 at 12:09

**Comment by GoogleCodeExporter** [comment]
```
This is now implemented in 0.5.2.
```

Original comment by `matt.rasmus@gmail.com` on 14 Apr 2009 at 1:33
- Changed state: **Done**
